src/uwtools/config.py

```diff
@@ -104,23 +106,47 @@ def dump_file(self, output_path):
         with open(output_path, 'w', encoding="utf-8") as file_name:
             yaml.dump(self.data, file_name, sort_keys=False)
 
+    class Loader(yaml.SafeLoader): #pylint: disable=too-many-ancestors
+        '''Yaml loader to add INCLUDE tag for yaml file concatenation'''
+
+        def __init__(self, stream: IO) -> None:
+            try:
+                self._root = os.path.split(stream.name)[0]
+            except AttributeError:
+                self._root = os.path.curdir
+
+            super().__init__(stream)
+
+    def add_include(loader: Loader, node: yaml.Node) -> Any: #pylint: disable=no-self-argument
```

Comment:
The pythonic way of achieving this result is to decorate it with `@staticmethod` instead of having the linter ignore it.